src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,19 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        ImGui::Text("CPU Backend \nSelect your Backend.");
+        ImGui::RadioButton("Dynarmic", &host.cfg.cpu_backend, 0);
```

Comment:
Dynarmic is 1 please. It's a little bit confusing but Unicorn = 0, Dynarmic=1 (please do not care about the enum value)

Reply:
oh wait, i have thinked is 0 like it said me ^^